web/src/routes.js

```diff
@@ -6,10 +6,10 @@ import PoC from './components/PoC';
 import Login from './containers/Login';
 
 const routes = [
-  { path: '/', component: PoC, exact: true, nonPrivate: true },
+  { path: '/', component: PoC, exact: true, nonPrivate: false },
```

Comment:
You have to be authenticated in order to send the API requests added in this PR, so...  there's this.